gouttelette/cmd/refresh_modules.py

```diff
@@ -1115,7 +1115,18 @@ def init_resources(paths: str) -> Dict:
 def generate_amazon_cloud(args: Iterable):
     module_list = []
 
-    for type_name in RESOURCES:
+    modules_file_path = os.path.abspath(
+        os.path.join(
+            os.path.dirname(__file__), "..", "config/amazon_cloud/modules.yaml"
```

Comment:
The path to modules.yaml will be provided by the user. If not provided, the default path (gouttelette/config/<collection>) will be taken. I do not think `os.path.dirname(__file__), "..", "config/amazon_cloud/modules.yaml"` will give the right path.